test/unit/component/footer.nuxt.spec.ts

```diff
@@ -0,0 +1,9 @@
+import {it, expect} from "vitest"
+import {mountSuspended} from "vitest-environment-nuxt/utils"
```

Comment:
use `"import {mountSuspended} from "@nuxt/test-utils/runtime"` since we migrated to `@nuxt/test-utils`